packages/charting/configure/src/charting-config.jsx

```diff
@@ -128,6 +128,7 @@ export class ChartingConfig extends React.Component {
       authorNewCategoryDefaults,
       chartingOptions,
       mathMlOptions = {},
+      charactersLimit,
```

Comment:
based on the request on pie-lib side, I suggest to rename this to `labesCharactersLimit`

Reply:
Done